src/backend/joanie/tests/core/test_api_admin_order_group.py

```diff
@@ -0,0 +1,294 @@
+"""
+Test suite for OrderGroup Admin API.
+"""
+
+from operator import itemgetter
+
+from django.test import TestCase
+
+from joanie.core import factories, models
+
+
+class OrderGroupAdminApiTest(TestCase):
+    """
+    Test suite for OrderGroup Admin API.
+    """
+
+    base_url = "/api/v1.0/admin/products"
+
+    # list
+    def test_admin_api_order_group_read_list_anonymous(self):
+        """
+        Anonymous users should not be able to list order groups.
+        """
+
+        product = factories.ProductFactory()
+        response = self.client.get(f"{self.base_url}/{product.id}/order-groups/")
+        self.assertEqual(response.status_code, 401)
+        content = response.json()
+        self.assertEqual(
+            content["detail"], "Authentication credentials were not provided."
+        )
+
+    def test_admin_api_order_group_read_list_authenticated(self):
+        """
+        Authenticated users should be able to list order groups.
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+
+        product = factories.ProductFactory()
+        order_groups = factories.OrderGroupFactory.create_batch(3, product=product)
+        factories.OrderGroupFactory.create_batch(5)
+
+        with self.assertNumQueries(7):
+            response = self.client.get(f"{self.base_url}/{product.id}/order-groups/")
+        self.assertEqual(response.status_code, 200)
+        content = response.json()
+        expected_return = [
+            {
+                "id": str(order_group.id),
+                "nb_seats": order_group.nb_seats,
+                "product": str(product.id),
+                "is_active": order_group.is_active,
+                "nb_availabilities": order_group.nb_seats
+                - order_group.get_nb_binding_orders(),
+                "created_on": order_group.created_on.isoformat().replace("+00:00", "Z"),
+            }
+            for order_group in order_groups
+        ]
+        self.assertEqual(content["count"], len(order_groups))
```

Comment:
```suggestion
        self.assertEqual(content["count"], 3)
```

IMO this should not be computed. You explicitly created 3 order groups and you expect the same number.